azext_iot/tests/dps/test_iot_dps_int.py

```diff
@@ -596,6 +596,95 @@ def test_dps_enrollment_group_lifecycle(self):
             )
         )
 
+    def test_dps_enrollment_group_symmetrickey_lifecycle(self):
```

Comment:
What about adding a test (unit is fine too) that exercises the  raised CLIError if the attestation type is not symmetricKey?

Reply:
Added to test above where an enrollment group with a certificate is created.

Reply:
Added to test above which already creates an enrollment group with a cert